WizardPanels/utils.js

```diff
@@ -0,0 +1,198 @@
+import {useRef, useState, useCallback, Children} from 'react';
```

Comment:
Please move it to the internals/Panels/ and make it to be used in both quickGuidePanels and WizardPanels.

Reply:
I moved files to internal/Panels.